### PR TITLE
add monorepo for react

### PR DESCRIPTION
**Pull request by januarsal-kindi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-  fe:
+  digiv-ui:
     container_name: digivapp-fe
-    image: digivapp.fe:latest
+    image: digivfe/digiv-ui:latest
     ports:
       - 80:80
-    
\ No newline at end of file
+    digiv-ui:
```
Comment (Asprilla24) [inline docker-compose.yml]: can you please change the services name to digiv-ui-chat or something, to match the actual services name

File: digiv-ui/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
+FROM nginx:1.18-alpine
```
Comment (Asprilla24) [inline digiv-ui/Dockerfile:1]: I think you need to remove this Dockerfile as you already defined the Dockerfile in the root folder

File: digiv-ui/.dockerignore [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.git
```
Comment (Asprilla24) [inline digiv-ui/.dockerignore:1]: Please remove this file too

File: digiv-ui/Dockerfile (hunk 2 above)
Comment (januarsal-kindi) [reply digiv-ui/Dockerfile]: fixed

File: digiv-ui/.dockerignore (hunk 3 above)
Comment (januarsal-kindi) [reply digiv-ui/.dockerignore]: fixed,thanks

**Review by Asprilla24 — APPROVED** [review]

**Pull request merged**